### PR TITLE
Twist 436

### LINKED ISSUE #436: Use Twist Commands to make the Rover's mobility compatible with the rest of the ROS framework
This will allow us to use external tools like keyboard_teleop, Gazebo and more

### PR DESCRIPTION
**Pull request by MartensCedric**
# Assignee Section

## Description
The drive controls have been redesigned with ROS twist commands. This means the rover can work with Gazebo. Unit tests have been added to test this functionality.

### Steps for Testing
Local Testing

1. rosgui, startgui
2. Open Gazebo with the Husky (You might need to set it up : http://wiki.ros.org/husky_gazebo/Tutorials/Simulating%20Husky
3. Go to the rover page
4. Move the rover around WASD

#### Rover Testing
Do a GIT, if it works, then this works. Make sure you test forward, backward and turning on the spot.

closes #436 
The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.

- [ ] Local Test Performed Successfully
- [ ] [General Integration Test](https://docs.google.com/document/d/1ug0CpA1cIzURP8DDFSvCt2CEJJSwJ6Ta6B1LG_hYk6I/edit) Performed Successfully

For Pull Requests that do not include code changes, it is not required to perform the tests above.

### CODE REVIEW

**Review by RealWilliamWells — COMMENTED** [review]
Looks good so far! It's definitely lot cleaner than the old drive controls

**Comment by MartensCedric** [comment]
> Looks good so far! It's definitely lot cleaner than the old drive controls

it doesnt work yet lol but soon :sunglasses: (hence the draft PR)

File: robot/basestation/static/js/pages/rover.js [hunk 1]
```diff
@@ -39,6 +8,115 @@ function printCommandsList () {
...
+   * with what values a Twist command should be sent.
+   */
+  function update_movement()
+  {
+    //todo: remove todos before PR.
+
+    //todo: put all this into a new javascript file called rover_controls.js
+
+    let desiredLinearSpeed = 0.5; // todo speed should based on throttle
+    let desiredAngularSpeed = 1; // todo angular speed should be based on throttle
```
Comment (AFknUsername) [inline robot/basestation/static/js/pages/rover.js:100]: 💢

**Review by AFknUsername — CHANGES_REQUESTED** [review]

**Comment by AFknUsername** [comment]
In your PR desc you wrote (to edit to explain)
:anger:

File: tests/unit/drive_controls_test.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from robot.rospackages.src.mcu_control.scripts.DriveControls import twist_to_rover_command, accelerate_value
+import robot.rospackages.src.mcu_control.scripts.DriveControls as dc
```
Comment (AFknUsername) [inline tests/unit/drive_controls_test.py:2]: Can't you flip these two lines and then do `from dc import ...`?

Comment (MartensCedric) [reply tests/unit/drive_controls_test.py:2]: nope, does not work

**Review by VimFreeman — APPROVED** [review]
Approving cause cedric forced me to

**Pull request merged**